reference/api/tasks.md

```diff
@@ -87,9 +93,9 @@ This value is always `null` for `dumpCreation` tasks.
 
 #### `indexDeletion`
 
-| Name                   | Description                                                                                       |
-| :--------------------- | :------------------------------------------------------------------------------------------------ |
-| **`deletedDocuments`** | Number of deleted documents. This should equal the total number of documents in the deleted index |
+| Name                   | Description                                                                                                                                                  |
+| :--------------------- | :----------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| **`deletedDocuments`** | Number of deleted documents. This should equal the total number of documents in the deleted index. `null` when the task status is `enqueued` or `processing` |
```

Comment:
If you like the suggestion of using "while" instead of "when", replace it here too.